src/main/java/view/OutputView.java

```diff
@@ -0,0 +1,84 @@
+package view;
+
+import enums.LottoRanking;
+import domain.Lotto;
+import domain.Lottos;
+
+import java.util.List;
+import java.util.Map;
+
+public class OutputView {
```

Comment:
static 클래스(유틸 클래스) 라면 객체가 생성되지 않도록 생성자를 `private` 접근제어자로 닫아두는것이 좋아요!

미래의 자신 또는 다른 개발자가 실수로 객체를 생성하는 일이 없게 하는것이죠!

```suggestion
public class OutputView {

    private OutputView() { }
```

Reply:
좋은 피드백 감사합니다! private으로 접근을 제한해 두는 연습을 길러야겠네요!